python/comps.py

```diff
@@ -130,7 +180,9 @@ def _get_top_n_comps(
 
 
 @nb.njit(fastmath=True)
-def _insert_at_idx_and_shift(arr, elem, idx):
+def insert_at_idx_and_shift(
```

Comment:
I'm removing the leading underscore from this function to indicate that it's no longer private, since we unit test it in the test suite below.

Reply:
Interesting, I wasn't aware of this convention. Was the underscore signaling that the function's use is recommended to not be used outside of the script ?

Reply:
Yup! Some context in [this SO answer](https://stackoverflow.com/a/1547163). Python doesn't have true private functions, since the interpreter won't block a caller from accessing an object whose name contains a leading underscore, but the convention is intended to warn code consumers that the behavior of that object is not guaranteed and should not be relied upon by consuming code.